src/loan_form/loan_calculate/Calculation.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Calculation({ total_months }) {
+  return (
+    <div>
+      <p>It will take {total_months == Infinity ? 'literally forever' : `${total_months} month${total_months == 1 ? '' : 's'}` } to pay off your loans.</p>
```

Comment:
👍 great copy

Reply:
oh also you should probably use `===` (identity equality) here since we don't want to typecast the `total_months` var which `==` will do.